common/static/js/vendor/jquery.leanModal.js

```diff
@@ -0,0 +1,61 @@
+// leanModal v1.1 by Ray Stone - http://finelysliced.com.au
+// Dual licensed under the MIT and GPL
+
+// Updated to prevent divs with duplicate IDs from being rendered.
+
+(function($) {
+    $.fn.extend({
+        leanModal: function(options) {
+            var defaults = {
+                top: 100,
+                overlay: 0.5,
+                closeButton: null
+            };
+
+            // Only append the overlay element if it isn't already present.
+            if ($("#lean_overlay").length == 0) {
```

Comment:
Right, JQuery always returns an array of matching elements, so it is just empty. See:

http://stackoverflow.com/questions/1339497/how-to-judge-whether-selected-result-is-empty-or-not-with-jquery